packages/terra-form-select/src/ComboboxField.jsx

```diff
@@ -79,6 +79,10 @@ const propTypes = {
    * Callback function triggered when the select value changes. function(value)
    */
   onChange: PropTypes.func,
+  /**
+   * [Deprecated] Placeholder text.
```

Comment:
There should be more info here, like this is no longer getting used at all and is "dead code"

Reply:
Update here https://github.com/cerner/terra-core/pull/3216/commits/49bc759de1137c07ad26355bca47f723c43ba011.